### DISCUSSION TITLE
How to add a horizontal scrollbar

### DISCUSSION THREAD

**Discussion by PMinkova** [wojtekmaj/react-pdf#1758] [Q&A]
Hi, everyone! I am trying to add a horizontal scrollbar on zoom in. Currently I have only a vertical one:  return (

        <Box id={`${id}-pdf`} display="flex" flexDirection="column" alignItems="center" overflow="auto" maxHeight="80vh">
            <Document file={fileUrl} onLoadSuccess={onDocumentLoadSuccess} onLoadError={console.error}>
                <div style={{ width: '100%', overflow: 'auto', maxHeight: '100%' }}>
                    <Page pageNumber={pageNumber} renderTextLayer={false} renderAnnotationLayer={false} scale={scale} />
                </div>
            </Document>
            <Box>
                <Button onClick={handleZoomIn}>Zoom In</Button>
                <Button onClick={handleZoomOut}>Zoom Out</Button>
            </Box>
            {pagesCount > 1 && <Pagination count={pagesCount} page={pageNumber} onChange={(_, page: number) => setPageNumber(page)} />}
        </Box>
    );

**Comment by Dealibros** [comment]
Any solution to this?
I would like as well to display the pages horizontally instead of a long list of pages as it is appearing now.

**Comment by wojtekmaj** [comment]
Are you missing maxWidth by any chance?
